out/extension.js

```diff
@@ -291,6 +319,33 @@ function handleSplunkFile(context) {
     specConfigs[currentDocument] = specConfig
 }
 
+async function handleSpl2File(context, progressBar) {
+    if (spl2Client) {
+        // Client and server are already running, try refreshing for case of new document
+        const range = new vscode.Range(new vscode.Position(0, 0), new vscode.Position(0, 1));
+        const text = vscode.window.activeTextEditor.document.getText(range) || " ";
+        vscode.window.activeTextEditor.edit((editBuilder) => {
+            // To refresh language server make a harmless edit by replacing the first character
```

Comment:
Just for my understanding, we're replacing the first character of the SPL2 document, correct? Do we need to replace it back with the original character?

Reply:
yeah we replace it with itself :) line 326 grabs the current text for the first character (or for a blank doc adds a space because we have to replace something for this to work) and then line 329 replaces that first character with itself